### PR TITLE
Feat checkout/AddressBook

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of 

- AddressBook Component for managing address
- checkout page

### CODE REVIEW

**Review by MaruthiPothuganti — APPROVED** [review]
Clean and well written code. Checkout feature working as specified. PR looks good to merge.

File: src/components/AddressBook/AddressCard/AddressCard.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <>
+      <div className="">
+        <p className="">
+          <span className="">{address.name}</span>
+          <span className="">{address.mobile}</span>
+        </p>
+        <p className="">
+          {(address.address, address.city, address.state)} -{" "}
+          <span className="">{address.pincode}</span>
```
Comment (harshitadaswani) [inline src/components/AddressBook/AddressCard/AddressCard.jsx:11]: can remove empty class names

File: src/components/AddressBook/AddressForm/AddressForm.jsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+  const updateAddress = (e) => {
+    setFormState((state) => ({ ...state, address: e.target.value }));
+  };
+  const updateCity = (e) => {
+    setFormState((state) => ({ ...state, city: e.target.value }));
+  };
+  const updateState = (e) => {
+    setFormState((state) => ({ ...state, state: e.target.value }));
+  };
```
Comment (harshitadaswani) [inline src/components/AddressBook/AddressForm/AddressForm.jsx:28]: Liked the use of functions here to ease the work

File: src/components/AddressBook/address-book.jsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            <AddressForm
+              initialFormState={initialFormState}
+              toggleIsAddressFormActive={toggleIsAddressFormActive}
+            />
+          </div>
+        </label>
+      )}
+    </>
+  );
+};
```
Comment (harshitadaswani) [inline src/components/AddressBook/address-book.jsx:190]: can separate out as an component

File: src/pages/Checkout/checkout.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  border: var(--size-16);
+}
+.card-action {
+  padding: var(--size-8);
+}
+.card-img-border {
+  border-radius: var(--size-16);
+}
+.card-cost {
+  font-size: var(--size-24);
```
Comment (harshitadaswani) [inline src/pages/Checkout/checkout.css:44]: liked the use of CSS variables here

**Review by harshitadaswani — APPROVED** [review]
Everything looks good! Liked the use of functions and CSS variables for sizes

File: src/components/AddressBook/AddressCard/AddressCard.jsx (hunk 1 above)
Comment (abhij1607) [reply src/components/AddressBook/AddressCard/AddressCard.jsx:11]: thanks for the suggestion, will remove them during refactor

File: src/components/AddressBook/address-book.jsx (hunk 3 above)
Comment (abhij1607) [reply src/components/AddressBook/address-book.jsx:190]: thanks for the suggestion

**Pull request merged**